ip.go

```diff
@@ -57,9 +57,17 @@ type IpAddressCommon struct { //nolint:golint
 	Project       Href        `json:"project"`
 	Global        bool        `json:"global_ip"`
 	Tags          []string    `json:"tags,omitempty"`
+	ParentBlock   ParentBlock `json:"parent_block,omitempty"`
 	CustomData    interface{} `json:"customdata,omitempty"`
 }
 
+// ParentBlock is the network block for the parent of an IP address
+type ParentBlock struct {
+	Network       string      `json:"network"`
+	Netmask       string      `json:"netmask"`
+	CIDR          int         `json:"cidr"`
+}
```

Comment:
According to the spec, there is also an Href available here, perhaps offered when not included.
https://github.com/t0mk/gometal/blob/main/v1/model_parent_block.go#L23

I've got hopes for Href being more valuable in the future: https://github.com/packethost/packngo/pull/280/files

Reply:
Updated.